web/client/components/share/ShareEmbed.jsx

```diff
@@ -56,7 +56,7 @@ class ShareEmbed extends React.Component {
     };
 
     render() {
-        const codeEmbedded = "<iframe style=\"border: none;\" height=\"400\" width=\"600\" src=\"" + this.generateUrl(this.props.shareUrl) + "\"></iframe>";
+        const codeEmbedded = "<iframe allowfullscreen style=\"border: none;\" height=\"400\" width=\"600\" src=\"" + this.generateUrl(this.props.shareUrl) + "\"></iframe>";
```

Comment:
no need for this

Reply:
if I try to embed the iframe in a different HTML file without `allowfullscreen ` full screen does not work @MV88